### PR TITLE
Fix Linear Webhooks

### PR DESCRIPTION
**Pull request by seripap**
- Update HTTPoision to fix CA untrusted issues
- Update graphql builder to support nested lists
- Fix/re-enable linear Webhooks
- Added `get_webhook`- if syncs are enabled with a lingering Webhook, the enable will fail. Should bypass creating a new Webhooks and use whats available (not implemented)
- Some dev pipeline updates (Added makefile/docker-compose)

Partially addresses some items in #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Linear
+# Linear-sync
```
Comment (jtormey) [inline README.md:1]: Nit: Could you change this to `LinearSync`, to match the module naming and the naming on the site?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    volumes:
+      - linear-db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: linear_dev
+    ports:
+      - 5432:5432
+volumes:
+    linear-db:
```
Comment (jtormey) [inline docker-compose.yml:16]: Nice! Thanks for adding this.

File: lib/linear/api/linear_api.ex [hunk 3]
```diff
@@ -123,14 +123,38 @@ defmodule Linear.LinearAPI do
...
   def create_webhook(session = %Session{}, opts) do
+    resourceTypes = [resourceTypes: ["Comment","Issue"]]
+    opts = Keyword.merge(opts, resourceTypes)
     query = %Query{
       operation: :webhookCreate,
-      variables: [input: Keyword.take(opts, [:url, :teamId])],
+      variables: [input: Keyword.take(opts, [:url, :teamId, :resourceTypes])],
       fields: [:success, webhook: [:id, :enabled]]
     }
+    # Logger.debug("Webhook query: #{opts}")
```
Comment (jtormey) [inline lib/linear/api/linear_api.ex:133]: Do you mind removing the commented code here?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+.PHONY: deps
+
+install:
+	mix setup
+
+run:
+	iex -S mix phx.server
+
+deps:
+	mix deps.compile
```
Comment (jtormey) [inline Makefile:11]: I'd rather not introduce `make` into the project if that's alright.

**Review by jtormey — COMMENTED** [review]
Thanks for taking a look at this! Just had a couple small comments 🙂

**Comment by seripap** [comment]
updated 👍🏼

**Review by jtormey — APPROVED** [review]
Awesome, thanks!

**Pull request merged**